core/services/ocr2/models/models.go

```diff
@@ -6,3 +6,19 @@ type MercuryCredentials struct {
 	Username  string
 	Password  string
 }
+
+func (c *MercuryCredentials) GetUsername() string {
```

Comment:
2 things: first, where are these used - i don't see it them in this change. 2nd the idiom is to name these Username(), Password(), etc (i.e. remove `Get`) and make the fields private

Reply:
I didn't want to make the fields private because I didn't want to impact where the fields are currently used, hence the Get prefix. This code isn't going to be merged; it's an investigation into what it might look like if we were to extract shared functionality into separate repos, and right now I'm just focusing on what functions could be extracted, without focusing too much on dependent types (to simplify extraction)